person.rb

```diff
@@ -2,13 +2,14 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
```

Comment:
- [ ] Same here, Please add Getters and Setters (attr_accessor) for `@rentals`, not Getters only (attr_reader) as you did.